components/pages/access-required/styles.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { createStyles } from "@solid/lit-prism-patterns";
+
+export default function styles(theme) {
+  return createStyles(theme, [], {
+    title: {
```

Comment:
This style seems like something that belongs in prism, rather than as a one-off.

Reply:
Yes, would be better for reusability. Not quite sure how to name it though, any suggestions? Something along the lines of info-title? (To be reused for titles on more static pages, to convey some info to the users.)

Reply:
We already have some rules for titles in general, so didn't want to adjust [the default typography that was created based on the earlier review by Andrei](https://www.figma.com/file/W2o8rM7yu5WxuRAlAOukuG/Pod-Browser?node-id=204%3A503).

Reply:
I've seen `hero` used for this kind of top banner (like a bigger h1) in bootstrap.

Reply:
Introduced Hero element in Prism. If https://github.com/inrupt/prism/pull/95 is accepted, I'll make sure to build a new version, and update this PR.